conda/recipes/pylibcudf/meta.yaml

```diff
@@ -29,8 +29,8 @@ build:
     - SCCACHE_BUCKET
     - SCCACHE_IDLE_TIMEOUT
     - SCCACHE_REGION
-    - SCCACHE_S3_KEY_PREFIX=cudf-aarch64 # [aarch64]
-    - SCCACHE_S3_KEY_PREFIX=cudf-linux64 # [linux64]
+    - SCCACHE_S3_KEY_PREFIX=pylibcudf-aarch64 # [aarch64]
```

Comment:
I don't think this is quite right.
(I _think_ all the cudf_* family of packages that compile C/C++/CUDA share a s3 bucket).

Reply:
Okay I was following `cudf_kafka` and `dask_cudf` recipes. eg. [`cudf_kafka`](https://github.com/rapidsai/cudf/blob/45b20d135a290d5f14e291316e94674653f71737/conda/recipes/cudf_kafka/meta.yaml#L32)

Reply:
Ah I see, that makes sense, sorry about the noise!